src/stream/src/executor/managed_state/aggregation/extreme.rs

```diff
@@ -65,28 +68,22 @@ where
     /// it's top n.
     top_n: BTreeMap<(Option<A::OwnedItem>, ExtremePk), Datum>,
 
-    /// The actions that will be taken on next flush
-    flush_buffer: BTreeMap<(Option<A::OwnedItem>, ExtremePk), FlushStatus<Datum>>,
-
     /// Number of items in the state including those not in top n cache but in state store.
     total_count: usize,
 
     /// Number of entries to retain in memory after each flush.
     top_n_count: Option<usize>,
 
-    /// Data type of the sort column
-    data_type: DataType,
-
-    /// The keyspace to operate on.
-    keyspace: Keyspace<S>,
+    // TODO: Remove this phantom to get rid of S: StateStore.
+    _phantom_data: PhantomData<S>,
 
     /// The sort key serializer
+    /// TODO: Remove this soon.
     serializer: ExtremeSerializer<A::OwnedItem, EXTREME_TYPE>,
```

Comment:
Let's store the pk length and it seems we can remove this now. Then the whole module can be cleaned up as well.